hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This release ensures that functions passed to SearchStrategy.flatmap return a SeachStrategy
```

Comment:
The existence of `SearchStrategy` is only public to help type hints, so I'd rephrase this a bit - and say thanks!  It's a bit awkward to thank yourself, so suggestion below, along with docs hyperlink syntax.

```rst
RELEASE_TYPE: patch

This patch improves the error message if the function ``f`` in 
:ref:`s.flatmap(f) <flatmap>` does not return a strategy.

Thanks to Kai Chen for this change!
```